build-tools/automation/azure-pipelines.yaml

```diff
@@ -17,14 +17,23 @@ resources:
 
 # Global variables
 variables:
-  InstallerArtifactName: installers
-  TestAssembliesArtifactName: test-assemblies
-  NUnitConsoleVersion: 3.9.0
-  DotNetCoreVersion: 2.1.701
-  HostedMacMojave: Hosted Mac Internal Mojave
-  HostedMac: Hosted Mac Internal
-  HostedWinVS2019: Hosted Windows 2019 with VS2019
-  VSEngWinVS2019: VSEng-MicroBuildVS2019
+  - name: InstallerArtifactName
```

Comment:
Is this *really* "beautifying"?  There's a certain elegance and simplicity to what you're replacing...

Reply:
Hah, by beautifying, I mean the commit history :P

I had to change the syntax to this in order to reference the variable group at the end for the keychain stuff.

Reply:
This syntax change is needed in order to use the following variable group:
```yaml
 - group: Xamarin-Build-Secrets
```

However, we could add this variable group to the build pipeline web UI, to avoid having to change the syntax for all other global variables. The majority of our secrets are coming from variable groups configured in the [build pipeline UI](https://devdiv.visualstudio.com/DevDiv/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=11410&view=Tab_Variables), I'd rather continue to maintain global variable groups there.

Reply:
Resolved